packages/components/plh/module-list-item/module-list-item.component.scss

```diff
@@ -30,8 +30,10 @@ $highlighted-color: var(--module-list-item-color-highlighted, var(--ion-color-se
 
   &[data-locked~="true"] {
     cursor: not-allowed;
-    border-color: var(--ion-color-gray-200);
+    border-color: var(--ion-color-primary-100);
     background-color: white;
+    border-width: 1px;
+    box-shadow: 0 2px 8px rgba(219, 227, 242, 0.6);
```

Comment:
nit(blocking): We don't want to have a hardcoded colour in the default component styling. As it's hard to manipulate SCSS var colour values, probably best to stick with `var(--ion-color-primary-100)` for the box shadow colour. You could always additionally change the opacity for the whole element (and adjust the colour values accordingly).